beacon-chain/sync/verify/blob.go

```diff
@@ -17,9 +17,11 @@ var (
 
 // BlobAlignsWithBlock verifies if the blob aligns with the block.
 func BlobAlignsWithBlock(blob blocks.ROBlob, block blocks.ROBlock) error {
-	if block.Version() < version.Deneb {
+	blockVersion := block.Version()
+	if !(version.Deneb <= blockVersion && blockVersion < version.Fulu) {
```

Comment:
would be easier to read if blockVersion was on the same side, but also this has the if statment flipped? is that right?

Reply:
I wish golang had this mathematical syntax:
```go
if !(version.Deneb <= blockVersion < version.Fulu) {
   ...
```

Python has it...

Reply:
I still think it's easier to read 
```
if version.Deneb > blockVersion || blockVersion >= version.Fulu
```